packages/oas-to-snippet/src/index.js

```diff
@@ -44,6 +44,12 @@ module.exports = (oas, operation, values, auth, lang, oasUrl) => {
     };
   }
 
+  if (lang === 'node') {
+    language.httpsnippet[2] = {
+      useObjectBody: true,
```

Comment:
You want to move this into `src/supportedLanguages` instead? Since it's not using `oasUrl` or `oas` like [node-simple](https://www.npmjs.com/package/api) snippets are we can probably leave this option contained there.

Reply:
Oh totally, way better. Didn't notice that config list before. Getting a new pr shortly.